docs/design-philosophy.md

```
@@ -200,6 +200,36 @@ hooks:
+
+- NG: `git checkout main`
+- OK: `git checkout {baseRefName}`
+
+`main` 以外をベースブランチとする PR（`develop`、`staging` 等）で誤動作する。
+（PR #20 修正コミット 05dd4969 より昇格）
+
+### グローバルスキル依存のフォールバック明示（SHOULD）
+
+スキルが他のグローバルスキル（`/commit` 等）に依存する場合、そのスキルが利用できない場合のフォールバック手順を SKILL.md に明記すること。
+
+- 例: `/commit スキル（グローバル定義）を使用するか、直接 `git commit -m "..."` を実行する。`
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)